apph-front/src/services/PhotoService.ts

```diff
@@ -39,16 +42,35 @@ export default class PhotoService {
       }
     };
     const successFunction = (val: string) => {
-      let i = 1;
       const tab: Array<ITable> = JSON.parse(val);
-      for (const elt of tab) {
-        elt.id = i++;
-      }
       handleSuccess(tab);
     };
     const errorFunction = (errorMessage: string) => {
       handleError(errorMessage);
     };
     return Server.request(URL, requestOptions, successFunction, errorFunction);
   }
+
+  static downloadImage(
+    id: number,
+    handleSuccess: (photo: IPhoto) => void,
+    handleError: (errorMessage: string) => void
+  ) {
+    const URL = `/photo/download`;
+    const userInfos = cookies.get('user');
+    const requestOptions = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        token: userInfos?.token
```

Comment:
```suggestion
        Authentication: userInfos?.token
```

Reply:
```suggestion
        Authorization: userInfos?.token
```
C'est une convention de nommage pour les token